pkg/controller/imagebuild/component/builddispatcher.go

```diff
@@ -278,9 +279,15 @@ func populateBuildStatus(obj *hephv1.ImageBuild, log logr.Logger, img v1.Image,
 	if err != nil {
 		log.Error(err, "Cannot calculate image labels", "imageName", imageName)
 	}
+	digest, err := img.Digest()
+	if err != nil {
+		log.Error(err, "Cannot retrieve image digest", "imageName", imageName)
+	}
 
 	log.Info(fmt.Sprintf("Final image size: %d", imageSize))
+
 	obj.Status.CompressedImageSizeBytes = strconv.FormatInt(imageSize, 10)
+	obj.Status.Digest = digest.String()
```

Comment:
Change is fine, just curious how this will be used?

Reply:
I was thinking we could use this to verify that the image produced by this ImageBuild and the one in the remote repository are actually the same image.  This would help for example when we go to delete the remote image to make sure its nothing else has pushed a newer one.

Reply:
Ok. Tags are immutable for Domino repos in AWS, but yes that won't be the case everywhere: 
https://github.com/dominodatalab/terraform-aws-eks/blob/882e01029ef85e276af000b2e1c21e3532d02850/modules/infra/submodules/storage/ecr.tf#L9

> This would help for example when we go to delete the remote image to make sure its nothing else has pushed a newer one.

When do we delete images?

Reply:
discussed in slack